app/src/main/java/com/davidhowe/chatgptclone/ui/textChat/TextChatViewModel.kt

```
@@ -0,0 +1,100 @@
+                    createdAt = System.currentTimeMillis(),
+                    isFromUser = false,
+                    content = "System response: ${System.currentTimeMillis()}",
+                ),
+                isProcessing = false
+            )
+        }
+    }
+
+    fun orderMessages() {
+
+    }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)